File: Day-15/Pattern22.js
function PatternTwentyTwo(n) {
  let i = 1;
  while (i <= n) {
    let str = "";
    let space = i - 1;
    while (space) {
      str += " ";
      space--;
    }
    let nums = n - i + 1;
    while (nums) {
      str += i;
      nums--;
    }
    i++;
    console.log(str);
  }
}

PatternTwentyTwo(6);

File: Day-15/Pattern22.py
n = int(input())

def PatternTwentyTwo(n):
    i = 1
    while(i <=n):
        space = i -1
        while(space):
            print(' ', end='')
            space -=1
        
        num = n-i +1
        while(num):
            print(i, end='')
            num-=1

        i+=1
        print(end='\n')
    
PatternTwentyTwo(n)
